src/content/docs/configuration/sched-ext.md

```diff
@@ -122,13 +223,19 @@ Options:
           CPUs
 
       --prefer-smt-core
-          Use SMT logical cores before using other physcial cores in core compaction
+          Use SMT logical cores before using other physical cores in core compaction
 
       --no-freq-scaling
           Disable frequency scaling by scx_lavd
 ```
 
+### The use cases of these schedulers are quite similar... why is that?
+
+Mainly because they are (for now) multipurpose schedulers, meaning they adapt to many workloads even if they don't excel at all of them.
+
+In order to find out which one fits you best, there is no other shortcut than to test it yourself.
+
 ## GitHub
 
-- scx-scheds (Schedulers): https://github.com/sched-ext/scx
-- https://github.com/sched-ext/scx-kernel-releases
+- scx-scheds (Schedulers): <https://github.com/sched-ext/scx>
```

Comment:
I don't really understand why ur doing this for links. Can you elaborate?

Reply:
Is to follow markdown guidelines